### PR TITLE
update community extensions

### PR DESCRIPTION
**Pull request by JTeijema**
Add 2 files to gitignore, fix a type, done some formatting and added 2 community repos. Should be good like this, but didn't add https://github.com/BartJanBoverhof/asreview-cnn-hpo

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -72,6 +72,7 @@ instance/
 # Sphinx documentation
 docs/_build/
 doc/_build/
+docs/source/_build/
```
Comment (J535D165) [inline .gitignore:75]: Can you revert these changes and make a separate PR if this is relevant? This keeps the git history clean.

File: docs/source/extensions/overview_extensions.rst [hunk 2]
```diff
@@ -23,9 +23,11 @@ of the core project:
...
 - :doc:`extension_visualization`: Plotting functionality for state files produced by ASReview.
 - :doc:`extension_wordcloud`: Create a visual impression of the contents of datasets via a wordcloud.
-- ``asreview-statistics``: Tool to give some basic properties of a dataset, such as number of
-  papers, number of inclusions. `GitHub <https://github.com/asreview/asreview-statistics>`__
-- ``asreview-hyperopt``: Optimize the hyperparameters of the models in ASReview. `GitHub <https://github.com/asreview/asreview-hyperopt>`__
+- `ASReview-statistics <https://github.com/asreview/asreview-statistics>`__: Tool to give some basic properties of a dataset, such as number of
+  papers, number of inclusions. 
+- `ASReview-vocab-extractor <https://github.com/asreview/asreview-extension-vocab-extractor>`__: This extension adds two feature extractors that extract vocabulary and vector matrices during simulation phases.
+
+.. - ``asreview-hyperopt``: Optimize the hyperparameters of the models in ASReview. `GitHub <https://github.com/asreview/asreview-hyperopt>`__
```
Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:30]: @Rensvandeschoot what do you think? This reverts the previous change.

File: docs/source/extensions/overview_extensions.rst [hunk 3]
```diff
@@ -34,9 +36,21 @@ of the core project:
 Community-maintained extensions
 -------------------------------
 
-The following extensions are developed and maintained by the ASReview community:
+ASReview has support for community-maintained extensions, that enable you to seemlessly integrate 
+your own code with the ASReview framework. These extensions can extend the software with new
```
Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:40]:
```suggestion
your code with the ASReview framework. These extensions can extend the software with new 
```

File: docs/source/extensions/overview_extensions.rst [hunk 4]
```diff
@@ -34,9 +36,21 @@ of the core project:
 Community-maintained extensions
 -------------------------------
 
-The following extensions are developed and maintained by the ASReview community:
+ASReview has support for community-maintained extensions, that enable you to seemlessly integrate 
+your own code with the ASReview framework. These extensions can extend the software with new 
+classifiers, query strategies, balance strategies, and feature extraction techniques.
```
Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:41]: Make clear that it can extend subcommands, models, datasets? The classifiers, query strategies, balance strategies, and feature extraction techniques are part of models.

File: docs/source/extensions/overview_extensions.rst [hunk 5]
```diff
@@ -34,9 +36,21 @@ of the core project:
...
+
+A template for creating your own ASReview extension can be found here: `Template <https://github.com/asreview/template-extension-new-model>`__.
```
Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:43]:
```suggestion
A template for creating an ASReview extension can be found here: `Template <https://github.com/asreview/template-extension-new-model>`__.
```

Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:43]: I think I prefer to point to the documentation instead of the template. The template is limited compared what ASReview offers for developers. 

https://asreview.readthedocs.io/en/latest/API/extensions_dev.html

File: docs/source/extensions/overview_extensions.rst [hunk 6]
```diff
@@ -34,9 +36,21 @@ of the core project:
...
+
+The following extensions are developed and currently maintained by the ASReview community:
```
Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:45]:
```suggestion
The following extensions are developed and maintained by the ASReview community:
```

File: docs/source/extensions/overview_extensions.rst [hunk 7]
```diff
@@ -34,9 +36,21 @@ of the core project:
...
+
+* ASReview CNN classifier 
+    - This repository contains a extention for ASReview logo. This plugin adds a convolutional neural network (CNN) model, that has been shown to outperform classical algorithms.
```
Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:48]: "that has been shown to outperform classical algorithms" Is this true?

Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:48]: `This repository contains a extention for ASReview logo. ` ?

**Review by J535D165 — CHANGES_REQUESTED** [review]
Thanks for extending this page. Nice extensions. 

Can you apply hard wraps (80char)?

File: docs/source/extensions/overview_extensions.rst [hunk 8]
```diff
@@ -21,11 +21,15 @@ The following extensions are officially supported and were developed as part
...
-- ``asreview-hyperopt``: Optimize the hyperparameters of the models in ASReview. `GitHub <https://github.com/asreview/asreview-hyperopt>`__
+- :doc:`extension_visualization`: Plotting functionality for state files 
+  produced by ASReview.
+- :doc:`extension_wordcloud`: Create a visual impression of the contents of 
+  datasets via a wordcloud.
+- `ASReview-statistics <https://github.com/asreview/asreview-statistics>`__: 
+  Tool to give some basic properties of a dataset, such as number of papers, 
+  number of inclusions. 
+.. - `ASReview-vocab-extractor <https://github.com/asreview/asreview-extension-vocab-extractor>`__: This extension adds two feature extractors that extract vocabulary and vector matrices during simulation phases.
+.. - ``asreview-hyperopt``: Optimize the hyperparameters of the models in ASReview. `GitHub <https://github.com/asreview/asreview-hyperopt>`__
```
Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:32]: Commented? Shouldn't this be included?

Comment (JTeijema) [reply docs/source/extensions/overview_extensions.rst:32]: I figured its outdated and "❗Highly experimental❗". Ill include it

File: docs/source/extensions/overview_extensions.rst [hunk 9]
```diff
@@ -21,11 +21,15 @@ The following extensions are officially supported and were developed as part
...
-  papers, number of inclusions. `GitHub <https://github.com/asreview/asreview-statistics>`__
-- ``asreview-hyperopt``: Optimize the hyperparameters of the models in ASReview. `GitHub <https://github.com/asreview/asreview-hyperopt>`__
+- :doc:`extension_visualization`: Plotting functionality for state files 
+  produced by ASReview.
+- :doc:`extension_wordcloud`: Create a visual impression of the contents of 
+  datasets via a wordcloud.
+- `ASReview-statistics <https://github.com/asreview/asreview-statistics>`__: 
+  Tool to give some basic properties of a dataset, such as number of papers, 
+  number of inclusions. 
+.. - `ASReview-vocab-extractor <https://github.com/asreview/asreview-extension-vocab-extractor>`__: This extension adds two feature extractors that extract vocabulary and vector matrices during simulation phases.
```
Comment (J535D165) [inline docs/source/extensions/overview_extensions.rst:31]:
```suggestion
  number of inclusions. 

.. - `ASReview-vocab-extractor <https://github.com/asreview/asreview-extension-vocab-extractor>`__: This extension adds two feature extractors that extract vocabulary and vector matrices during simulation phases.
```

**Review by J535D165 — APPROVED** [review]

**Comment by J535D165** [comment]
Thanks!
